doc/Language/regexes.pod6

```diff
@@ -2017,6 +2020,11 @@ regexes. Just as with ordinary C<|> interpolation, the longest match succeeds:
     my @a = '2', 23, rx/a.+/;
     say ('b235' ~~ /  b @a /).Str;      # OUTPUT: «b23»
 
+You may use C<@(code)> to interpolate any expression that evaluates to a list:
+
+    my %h = a => 1, b => 2;
+    say S:g/@(%h.keys)/%h{$/}/ given 'abc';    # OUTPUT: «12c>
+
```

Comment:
What would they expand to? Values separated by |?

Reply:
Yes, this is mentioned in the previous paragraph. I clarified that @(code) is the same as first assinging code to @a and then interpolating @a.